.gitignore

```diff
@@ -0,0 +1,4 @@
+.directory
+__pycache__
+library_coverage_xml_and_fulltext_indicators.db*
```

Comment:
`library_coverage_xml_and_fulltext_indicators.db` still part of the diff. Can you run:

```
git rm --cached library_coverage_xml_and_fulltext_indicators.db
```

Reply:
Done!